docs/source/scripts/build_quantization_configs.py

```diff
@@ -0,0 +1,23 @@
+"""
+This script will generate default values of quantization configs.
+These are for use in the documentation.
+"""
+
+from torch.ao.quantization.fx.backend_config import get_native_backend_config_dict
+import os.path
+from pprint import pprint
+
+
+# Create a directory for the images, if it doesn't exist
+QUANTIZATION_CONFIG_IMAGE_PATH = os.path.join(
+    os.path.realpath(os.path.join(__file__, "..")),
+    "quantization_configs"
```

Comment:
maybe quantization_operator_config, or quantization_backend_config? since `quantization config` is already sued for QConfig..

Reply:
sure, changed